packages/main/src/MessageStrip.ts

```diff
@@ -151,10 +69,86 @@ const metadata = {
  * @public
  * @since 0.9.0
  */
+@customElement("ui5-message-strip")
+@languageAware
+/**
+ * Fired when the close button is pressed either with a
+ * click/tap or by using the Enter or Space key.
+ *
+ * @event
+ * @public
+ */
+@event("suggestion-scroll", {
+	close: {},
+})
 class MessageStrip extends UI5Element {
-	static get metadata() {
-		return metadata;
-	}
+	/**
+	 * Defines the component type.
+	 * <br><br>
+	 * <b>Note:</b> Available options are <code>"Information"</code>, <code>"Positive"</code>, <code>"Negative"</code>,
+	 * and <code>"Warning"</code>.
+	 *
+	 * @type {sap.ui.webc.main.types.MessageStripDesign}
+	 * @defaultvalue "Information"
```

Comment:
you need to add `@name sap.ui.webc.main.MessageStrip.prototype.{prop|slot}`
for the public props and slots, otherwise they won't make it to the api.json

this is one of the new things to keep in mind
in this case `@name sap.ui.webc.main.MessageStrip.prototype.design`